Core/include/Acts/Utilities/SpacePointUtility.hpp

```diff
@@ -0,0 +1,169 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/EventData/Measurement.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Geometry/TrackingGeometry.hpp"
+#include "Acts/SpacePointFormation/SpacePointBuilderConfig.h"
+#include "Acts/Utilities/Result.hpp"
+
+#include <array>
+#include <cstddef>
+#include <iostream>
+#include <memory>
+#include <vector>
+
+namespace Acts {
+
+/// @brief Storage container for variables related to the calculation of space
+/// points
+struct SpacePointParameters {
+  /// Vector pointing from bottom to top end of first SDE
```

Comment:
Can we get some more meaningful names here ?

Reply:
done.